quickjs.c

```diff
@@ -7388,20 +7427,23 @@ JSValue JS_GetProperty(JSContext *ctx, JSValue this_obj, JSAtom prop)
     return JS_GetPropertyInternal2(ctx, this_obj, prop, this_obj, NULL, FALSE);
 }
 
-static JSValue JS_GetPropertyInternalWithIC(JSContext *ctx, JSValue obj,
+static force_inline JSValue JS_GetPropertyInternalWithIC(JSContext *ctx, JSValue obj,
```

Comment:
Why the force_inline? (Also applies to other places where you added it; should be used sparingly)

Reply:
`JS_GetPropertyInternalWithIC` and `JS_SetPropertyInternalWithIC` are relatively high-frequency calling methods. force_inline brings about a nearly 2% improvement on my Apple M1 machine, and at the same time, they do not significantly increase the size of the program.